Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -1814,6 +1815,8 @@ bool Parser::is_parsing_svg_presentation_attribute() const
 // FIXME: URLs shouldn't be completed during parsing, but when used.
 Optional<URL::URL> Parser::complete_url(StringView relative_url) const
 {
+    if (!m_url.is_valid())
+        return URL::Parser::basic_parse(relative_url);
     return m_url.complete_url(relative_url);
```

Comment:
Note that we're not *supposed* to complete URLs in the CSS parser. (Or at least, objects are supposed to know their original URL string verbatim.) That was a mistake a long time ago and it's just never been corrected.